client/src/client.rs

```diff
@@ -784,6 +784,52 @@ pub trait RpcApi: Sized {
         let args = [into_json(blockhash)?, into_json(timeout)?];
         self.call("waitforblock", &args)
     }
+
+    fn wallet_create_funded_psbt(
+        &self,
+        utxos: &[json::CreateRawTransactionInput],
+        outs: &HashMap<String, Amount>,
+        locktime: Option<i64>,
+        options: Option<json::WalletCreateFundedPsbtOptions>,
+        bip32derivs: Option<bool>,
+    ) -> Result<json::WalletCreateFundedPsbtResult> {
+        let outs_converted = serde_json::Map::from_iter(
+            outs.iter().map(|(k, v)| (k.clone(), serde_json::Value::from(v.as_btc()))),
+        );
+        let args = [
+            into_json(utxos)?,
+            into_json(outs_converted)?,
+            opt_into_json(locktime)?,
+            into_json(options)?,
+            into_json(bip32derivs)?,
+        ];
+        self.call("walletcreatefundedpsbt", &args)
```

Comment:
You should be using `handle_defaults` here like I elaborated in another comment.